easybuild/easyconfigs/t/Tk/Tk-8.6.7-foss-2017b.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'Tk'
+version = '8.6.7'
+
+homepage = 'http://www.tcl.tk/'
+description = """Tk is an open source, cross-platform widget toolchain that provides a library of basic elements for
+ building a graphical user interface (GUI) in many different programming languages."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = ["http://prdownloads.sourceforge.net/tcl"]
+sources = ['%(namelower)s%(version)s-src.tar.gz']
+checksums = ['061de2a354f9b7c7d04de3984c90c9bc6dd3a1b8377bb45509f1ad8a8d6337aa']
+
+patches = ['Tk-8.6.4_different-prefix-with-tcl.patch']
```

Comment:
@Helios07 style remark: please move this up, and also include a SHA256 checksum for the patch (`--include-checksums` will do that for you)